src/Button/ButtonClose.js

```diff
@@ -0,0 +1,44 @@
+import {COMMON, LAYOUT} from '../constants'
+import theme from '../theme'
+import sx from '../sx'
+import {XIcon} from '@primer/octicons-react'
+import React, {forwardRef} from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+const StyledButton = styled.button`
+  border: none;
+  padding: 0;
+  background: transparent;
+  outline: none;
+  cursor: pointer;
+
+  &:focus {
+    box-shadow: 0 0 0 3px rgba(139, 148, 158, 0.3);
```

Comment:
Just curious, where'd these focus styles come from?

Reply:
Hmm good question - I copied over the focus box shadow used in the close button in toasts, but those are on a dark background in light mode, which is why its grey. I would probably be better to used the primary box-shadow-focus here instead: https://github.com/primer/css/blob/master/src/support/variables/misc.scss#L23

Reply:
The official Primer `.button-close` doesn't include a focus outline, but it probably should